containers/vars/defaults.yaml

```diff
@@ -64,5 +68,15 @@ images:
         - "git+https://github.com/pulp/pulp_maven.git"
         - "git+https://github.com/pulp/pulp_python.git"
         - "git+https://github.com/pulp/pulp_rpm.git"
+  - pulp_web_stable:
```

Comment:
FYI, if the variable format for this file is too complex or lacking necessary features, feel free to rework it entirely.

In the past, we wrote out custom variables and used the pulp-operator repo on our template for plugins' CI. But not anymore.

Reply:
ok. A future TODO.